backend/src/main/scala/com/softwaremill/bootzooka/api/RoutesSupport.scala

```diff
@@ -101,3 +105,11 @@ trait CacheSupport {
     extensionsTest("png", "svg", "gif", "woff", "jpg").flatMap { _ => cacheResponse } |
       doNotCacheResponse
 }
+
+trait SecuritySupport {
+  val addSecurityHeaders = respondWithHeaders(
```

Comment:
maybe it would make sense to add the link to OWASP that you mentioned? So that it's clear where this comes from